src/modules/commander/commander_params.c

```diff
@@ -984,3 +984,18 @@ PARAM_DEFINE_INT32(COM_POWER_COUNT, 1);
  * @decimal 3
  */
 PARAM_DEFINE_FLOAT(COM_LKDOWN_TKO, 3.0f);
+
+/**
+ * Delay between RC loss and configured reaction
+ *
+ * A non-zero, positive value makes the failsafe reaction first stop the vehicle and wait
+ * before proceeding with the configured failsafe reaction NAV_RCL_ACT.
+ * A zero or negative value disables the delay.
+ *
+ * @group Commander
+ * @unit s
+ * @min -1.0
+ * @max 25.0
+ * @decimal 3
+ */
+PARAM_DEFINE_FLOAT(COM_LL_DELAY, 15.0f);
```

Comment:
Something like this perhaps?

```suggestion
PARAM_DEFINE_FLOAT(COM_RC_LOSS_DLAY, 15.0f);
```

We should probably finally transition `NAV_RCL_ACT` as well.

Reply:
I suppose it's more like COM_RC_LOSS_ACTION_DELAY...

Reply:
How about `COM_RCL_ACT_T` ? I switched in 04029dd